assets/js/components/SapSystemDetails/GenericSystemDetails.jsx

```diff
@@ -12,6 +12,9 @@ import {
   systemInstancesTableConfiguration,
 } from './tableConfigs';
 
+export const renderEnsaVersion = (ensaVersion) =>
```

Comment:
I'm not sure if the place for this function is the best one.
Suggestions?

Reply:
You can both leave it here or have a dedicated file, if we maybe get to have more and more render functions we can have something like `renderers.js` or something like that. For now I think we can leave it